### DISCUSSION TITLE
Czech translation

### DISCUSSION THREAD

**Discussion by pavelsolar** [TandoorRecipes/recipes#393] [General]
Hello Vabene1111,
I'm currently in love with your Recipes app and I'd like to participate on translation to Czech language. My dev skills are pretty limited, but eventually I can get things done :) Is it possible to translate just django.mo and djnago.po files? Or do you prefer any other method using translation pairing tables? When done, how can I get my translation to you so it can be included in next version release? Thank you very much and keep up the good work!!

Best regards. Pavel

**Comment by aarondoet** [comment]
You can contribute translations by creating a transifex account and then joining the group. The link to the group and maybe some more information can be found here: https://vabene1111.github.io/recipes/contribute/#contribute-translations

**Reply by pavelsolar** [reply]
@l0c4lh057 That's great! I'll look into it later today. Thanks

**Reply by pavelsolar** [reply]
@l0c4lh057 OK, so I've requested a Czech translation on transifex and now I'll wait fo someone to accept it. Is that correct?

**Reply by aarondoet** [reply]
Yes, you now have to wait for vabene1111 to accept it, I don't think there is anyone else who can do that.

**Reply by vabene1111** [reply]
yes, i am not very happy with the translation system at the moment but its the best i currently have. 
Sometimes (like this time) transifex just does not send me any notifications

I have acccepted the language requests, now you will have to make a new request to actually join the newly create language as its not possible for me to add you easily for some reason 🙈

**Reply by pavelsolar** [reply]
@vabene1111  Cool, I've sent you another request on Transifex. Initial process is a bit strange, but understandable. When would it be possible to include translation in next release?

**Reply by vabene1111** [reply]
i have accepted you request, the next release is scheduled to be soon but it should be enough time for you to finish the translations. 

Thank you for helping and for the kind words 👍

**Reply by pavelsolar** [reply]
@vabene1111 Thank you for accepting my invitation. I'll start working on it right away!

**Reply by pavelsolar** [reply]
@vabene1111 My translations on transifex are complete. If possible, please include them in your next version release so I can then do some debugging. Iľl keep you posted about some minor updates if any, but for now I'm done. Have a great weekend. Pavel

**Reply by vabene1111** [reply]
Thank you very much for the translations, whenever you change something transifex will create a new PR so if you find any issues just fix them in transifixe and i will get notified automatically.
